sdk/python/feast/repo_config.py

```diff
@@ -123,16 +82,23 @@ class RepoConfig(FeastBaseModel):
     """
 
     provider: StrictStr
-    """ str: local or gcp or redis """
+    """ str: local or gcp """
 
-    online_store: OnlineStoreConfig = SqliteOnlineStoreConfig()
+    online_store: Any
```

Comment:
So we can't use a common base class here?

Reply:
Yeah - there's no good way of expressing "Subclass of T". We could theoretically have:
```
    online_store: FeastConfigBaseModel
```

But since `FeastConfigBaseModel` has `extra='forbid'` that would break validation since each subclass will have fields no in `FeastConfigBaseModel`. You can have `extra='allow'` but than actually makes the validation of the individual config classes too loose.